appium-dotnet-driver/Appium/Windows/WindowsDriver.cs

```diff
@@ -0,0 +1,200 @@
+        public WindowsDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities)
+            : base(remoteAddress, SetPlatformToCapabilities(desiredCapabilities, Platform))
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the WindowsDriver class using the specified Appium local service and desired capabilities
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities of the browser.</param>
+        public WindowsDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities)
+            : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform))
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the WindowsDriver class using the specified remote address, desired capabilities, and command timeout.
+        /// </summary>
+        /// <param name="remoteAddress">URI containing the address of the WebDriver remote server (e.g. http://127.0.0.1:4723/wd/hub).</param>
+        /// <param name="desiredCapabilities">An <see cref="DesiredCapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
+        public WindowsDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
+            : base(remoteAddress, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the WindowsDriver class using the specified Appium local service, desired capabilities, and command timeout.
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
+        public WindowsDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
+            : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
+        {
+        }
+
+        #region IFindByWindowsUIAutomation Members
+
+        /// <summary>
+        /// Finds the first element that matches the Windows UIAutomation selector
+        /// </summary>
+        /// <param name="selector">UIAutomation selector</param>
+        /// <returns>First element found</returns>
+        public W FindElementByWindowsUIAutomation(string selector)
+        {
+            //return (W)this.FindElement("-windows uiautomation", selector);
+            throw new NotImplementedException();
```

Comment:
May be there is sence remove this string and make this string not commented

```C#
return return (W)this.FindElement("-windows uiautomation", selector);
```

It depends on server-side and when it would be available then it should work immediately